src/App.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { Route, Routes } from 'react-router-dom';
+import Home from './components/Home';
+import './App.css';
+import Layout from './components/Layout';
+import Signup from './components/Signup';
+import Login from './components/Login';
+import Details from './components/Details';
+import Reserve from './routes/Reserve';
+import MyReservations from './components/MyReservations';
+import AddMotorbikeForm from './components/AddMotorbikeForm';
+import DeleteMotorbike from './components/DeleteMotorbike';
+
+function App() {
```

Comment:
- [ ] Whenever I try to make a reservation, the first click on the button fails the request, the second click succeeds:

![image](https://github.com/SaneMyburg/motorbike-front-end/assets/57421823/715ea69e-2e84-4889-9231-7344748e4058)

This means, I need to click twice on the reservation button for It to work correctly.

Kindly, take a look at this issue so your users can reserve items on one click. 👍